site/layouts/partials/spread.html

```diff
@@ -15,11 +15,95 @@ <h2 class="section-heading">{{ . }}</h2>
+							<input type="hidden" name="bn" value="PP-BuyNowBF:btn_buynowCC_LG.gif:NonHostedGuest">    
+			 				<input type="hidden" name="on0" value="payment">                               
+                                  <div class="form-inline form-group quater-width">
+                                    <div class="input-group">
+                                      <select name="os0" id="monthlyamount" style="border-radius:5px;width: 200px" class="form-control">
+                                          <option value="monthly">$5/month</option>
+                                          <option value="monthly">$10/month</option>
+                                          <option value="monthly">$25/month</option>
+                                          <option value="monthly">$50/month</option>
+                                          <option value="monthly">$100/month</option>
+                                          <option value="monthly">$250/month</option>
+                                          <option value="monthly">$500/month</option>
+                                        </select>
+                                        <input type="hidden" name="currency_code" value="USD">
+							            <input type="hidden" name="option_select0" value="monthly">
+							            <input type="hidden" name="option_amount0" value="5.00">
+							            <input type="hidden" name="option_select1" value="monthly">
+							            <input type="hidden" name="option_amount1" value="10.00">
+							            <input type="hidden" name="option_select2" value="monthly">
+							            <input type="hidden" name="option_amount2" value="25.00">
+							            <input type="hidden" name="option_select3" value="monthly">
+							            <input type="hidden" name="option_amount3" value="50.00">
+							            <input type="hidden" name="option_select4" value="monthly">
+							            <input type="hidden" name="option_amount4" value="100.00">
+							            <input type="hidden" name="option_select5" value="monthly">
+							            <input type="hidden" name="option_amount5" value="250.00">
+							            <input type="hidden" name="option_select6" value="monthly">
+							            <input type="hidden" name="option_amount6" value="500.00">
+							            <input type="hidden" name="option_index" value="0">
+                                        </div>
+                                  </div>
+							<input type="image" src="https://www.paypalobjects.com/en_GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online!">
+							<img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
+						</form>							
+						</div>
+							<p class="lead">
+								The recommended way for donating to us is to join as a regular supporter. You can decide on if you want to pay 10 Euro or more monthly.		
```

Comment:
@kubami9 we can use Dollar here instead of Euro. @mariobehling can you specify the amount ?

Reply:
Sure, I'll do it with my next PR 😄